src/anomalib/deploy/inferencers/torch_inferencer.py

```diff
@@ -122,18 +94,7 @@ def pre_process(self, image: np.ndarray) -> Tensor:
         Returns:
             Tensor: pre-processed image.
         """
-        transform_config = (
-            self.config.dataset.transform_config.eval if "transform_config" in self.config.dataset.keys() else None
-        )
-
-        image_size = (self.config.dataset.image_size[0], self.config.dataset.image_size[1])
-        center_crop = self.config.dataset.get("center_crop")
-        if center_crop is not None:
-            center_crop = tuple(center_crop)
-        normalization = InputNormalizationMethod(self.config.dataset.normalization)
-        transform = get_transforms(
-            config=transform_config, image_size=image_size, center_crop=center_crop, normalization=normalization
-        )
+        transform = A.from_dict(self.metadata["transform"])
```

Comment:
This way, we instantiate the transforms object with every call. Maybe it would be better to load the transforms in the constructor and store them as `self.transform`